packages/lab/src/radio-button/RadioButtonGroup.tsx

```diff
@@ -1,38 +1,44 @@
 import { clsx } from "clsx";
 import {
   ChangeEventHandler,
-  ComponentType,
   forwardRef,
   HTMLAttributes,
-  useImperativeHandle,
-  useRef,
   useState,
 } from "react";
 import { makePrefixer, useId } from "@salt-ds/core";
-import { useFormFieldProps } from "../form-field-context";
-import { FormGroup } from "../form-group";
-import { FormLabel } from "../form-field";
 import { RadioGroupContext } from "./internal/RadioGroupContext";
-import { RadioButton } from "./RadioButton";
-import { RadioButtonIconProps } from "./RadioButtonIcon";
 
 import "./RadioButtonGroup.css";
 
 const withBaseName = makePrefixer("saltRadioButtonGroup");
 
+export type RadioButtonGroupDirectionProps = "horizontal" | "vertical";
+
 export interface RadioButtonGroupProps extends HTMLAttributes<HTMLDivElement> {
   className?: string;
+  /**
+   * Set the selected value when initialized.
+   */
   defaultValue?: string;
-  icon?: ComponentType<RadioButtonIconProps>;
-  legend?: string;
+  /**
+   * Set the group direction.
+   */
+  direction?: RadioButtonGroupDirectionProps;
+  /**
+   * Only for horizontal direction. When `true` the text in radio button label will wrap to fit within the container. Otherwise the radio buttons will wrap onto the next line.
+   */
+  labelWrap?: boolean;
```

Comment:
It should be the label. Are you sure you've set it to `true`?

Reply:
It works fine but I'm talking about the prop name and description. Is `labelWrap` the right choice? Is it better to describe the whole behaviour as controlling the wrapping of the whole group rather than just the labels?

Reply:
I see what you're saying. Maybe a `wrap` prop, default `true` should do. And when `false` then the label text would wrap